build-support/docker/bootstrap-docker-snapshot-image.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+CENTOS_SNAPSHOT_TAG='pants_centos6_snapshot'
```

Comment:
This is likely to lead to trouble and confusion, as you're overwriting the same tag with different versions, and it may be unclear which version you're currently using.  It's already bad enough that we use `:latest` in production, which makes our CI runs unreproducible.

Possibly we should generate a unique tag based on the git sha or something, and have the travis image depend on that tag (passed in via ARG for desktop use, or hard-coded for CI use). This solves a problem in both development runs and "production" CI usage.

Reply:
I like the idea of the second paragraph a lot (and the first) and will be able to look into it more later today.